src/vic-stdlib.c

```diff
@@ -44,6 +46,33 @@ void vic_get(void)
     }
 }
 
+void vic_list(void)
+{
+    char option = '\0';
+    char *args_format = "-%c ";
+
+    if (vic_args(args_format, &option) != 1) {
```

Comment:
@vargac pure `ls` should work as well. Therefore it needs to have the same functionality like `ls -f`